apps/discussions/package.json

```
@@ -62,7 +64,8 @@
     "lint": "solium --dir ./contracts",
     "lint:fix": "eslint . --fix && solium --dir ./contracts --fix",
     "coverage": "cross-env SOLIDITY_COVERAGE=true npm run ganache-cli:test",
-    "ganache-cli:test": "sh ./node_modules/@aragon/test-helpers/ganache-cli.sh"
+    "ganache-cli:test": "sh ./node_modules/@aragon/test-helpers/ganache-cli.sh",
```

This script will probably be located at the root folder as long as we use the hoisted version for test-helpers
```suggestion
    "ganache-cli:test": "sh ../node_modules/@aragon/test-helpers/ganache-cli.sh",
```